air/src/options.rs

```diff
@@ -170,23 +168,25 @@ impl From<ProvingOptions> for WinterProofOptions {
 // EXECUTION OPTIONS
 // ================================================================================================
 
+/// Duplicate of `miden_processor::fast::DEFAULT_CORE_TRACE_FRAGMENT_SIZE` until `ExecutionOptions`
+/// is moved to `miden_air`.
```

Comment:
is `miden_air` the right place?

Reply:
Oops, I meant "moved out of `miden_air`"